draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -357,19 +361,20 @@ values are:
 * RuleID = 22 (8-bit) for which SCHC compression was not possible
 
 The remaining RuleIDs are available for compression. RuleIDs are shared between
-uplink and downlink sessions.  A RuleID not FPortUp or FPortDown means that
-the fragmentation is not used, thus the packet SHOULD be send to C/D layer.
+uplink and downlink sessions.  A RuleID different from FPortUp or FPortDown
+means that the fragmentation is not used, thus the packet SHOULD be sent to C/D
+layer.
 
 The only uplink messages using the FPortDown port are the fragmentation SCHC
 control messages of a downlink fragmentation session (ex ACKs). Similarly, the
 only downlink messages using the FPortUp port are the fragmentation SCHC
 control messages of an uplink fragmentation session.
 
-Application can have multiple fragmentation sessions between a device and one
-or several SCHC gateways.  A set of FPort values is REQUIRED for each gateway
+An application can have multiple fragmentation sessions between a device and one
+or several SCHC gateways.  A set of FPort values is REQUIRED for each NGW
```

Comment:
Good catch, anyway it is not only NGW but NGW + SCHC F/R. Should we put it there or just let NGW so it is easier to read ?

Reply:
Thank you @Oliv4945 ! You are right. I think I should have put `SCHC gateway` here even if it repeats the previous sentence. On the other hand, looking at this again I am wondering if the reader can be mislead to think that there could be multiple simultaneous sessions between a device and one SCHC gateway, which would contradict [sec 5.6.1][1] if I am reading it correctly. In practice this text seems to suggests that we can have multiple SCHC datagram exchanges happening simultaneously, which for me could be admissible as long as we use different FPorts as is indicated in this section (I am not sure how practical it's going to be, but I don't see good reason to disallow it). Do you think we should clarify 5.6.1 or you would prefer something else?

[1]: https://acklio.github.io/schc-over-lorawan/ivo-comments-v03/draft-ietf-lpwan-schc-over-lorawan.html#rfc.section.5.6.1